### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Dhaiven**

### CODE REVIEW

**Review by CoolLoong — CHANGES_REQUESTED** [review]

File: src/main/java/cn/nukkit/level/GameRules.java [hunk 1]
```diff
@@ -76,28 +80,24 @@ public void refresh() {
...
     public void setGameRule(GameRule gameRule, float value) {
-        if (!gameRules.containsKey(gameRule)) {
-            throw new IllegalArgumentException("Gamerule does not exist");
-        }
-        gameRules.get(gameRule).setValue(value, Type.FLOAT);
-        stale = true;
+        setGameRule(gameRule, value, Type.FLOAT);
     }
 
     public void setGameRules(GameRule gameRule, String value) throws IllegalArgumentException {
```
Comment (CoolLoong) [inline src/main/java/cn/nukkit/level/GameRules.java:103]: rename to setGameRule

**Review by CoolLoong — CHANGES_REQUESTED** [review]

File: src/main/java/cn/nukkit/level/PlayerChunkManager.java [hunk 2]
```diff
@@ -40,9 +42,9 @@ public int compare(long chunkHash1, long chunkHash2) {
     };
     private final Player player;
     //保存着上tick已经发送的全部区块hash值
-    private final LongOpenHashSet sentChunks;
+    private final @Nonnull LongOpenHashSet sentChunks;
```
Comment (CoolLoong) [inline src/main/java/cn/nukkit/level/PlayerChunkManager.java:45]: Use @NotNull in jetbarin

**Review by CoolLoong — CHANGES_REQUESTED** [review]

File: src/main/java/cn/nukkit/level/GameRules.java (hunk 1 above)
Comment (Dhaiven) [reply src/main/java/cn/nukkit/level/GameRules.java:103]: it's already named setGameRule

Comment (CoolLoong) [reply src/main/java/cn/nukkit/level/GameRules.java:103]: rename setGameRules(GameRule gameRule, String value) to setGameRule

Comment (Dhaiven) [reply src/main/java/cn/nukkit/level/GameRules.java]: ok

**Review by CoolLoong — APPROVED** [review]

**Pull request merged**